common/views/components/VenueHours/VenueHours.js

```diff
@@ -175,12 +192,7 @@ const VenueHours = ({ venue, isInList }: Props) => {
             })}
           >
             {upcomingExceptionalPeriod && upcomingExceptionalPeriod.length > 0 && (
-              <JauntyBox
-                topLeft={randomPx()}
-                topRight={randomPx()}
-                bottomRight={randomPx()}
-                bottomLeft={randomPx()}
-              >
+              <JauntyBox ref={jauntyRef}>
```

Comment:
Is this worth compartmentalising?

Reply:
How do you mean?

Reply:
Component-alising I meant...

Reply:
aaaaha. To get around the need for `ref`s? Could do, yeah.